docs/help/specifications-support/asyncapi-support.md

```diff
@@ -16,29 +12,52 @@ Bump is able to extract:
+The following authentication types are **not** supported: 
+- `X509`
+- `symmetricEncryption`
+- `asymmetricEncryption`
+- `plain`
+- `scramSha256`
+- `scramSha512`
+- `gssapi.`
+To describe these authentication types, please use our [custom `x-topics` property](doc-topics.md) for now.
 
 ## readOnly and writeOnly properties
 
-JSON Schema provide possibility to declare a property as read or write Only, with boolean fields writeOnly and readOnly (cf JSON Schema documentation).
+:::info
+Properties defined as `writeOnly` are hidden when they belong to a **subscribe operation**.
+Properties defined as `readOnly` are hidden when they belong to a **publish operation**.
+:::
+
+In the AsyncAPI context, most often, a subscriber (or consumer) is an application that connects to the broker, manifests an interest in a certain type of message, and leaves the connection open so the broker can push messages from the publisher to them. (more info on that [in the AsyncAPI documentation](https://www.asyncapi.com/docs/tutorials/getting-started/event-driven-architectures))
+
+JSON Schema provides the possibility to declare a property as read or write only, with boolean fields `writeOnly` and `readOnly` (cf [JSON Schema documentation](https://json-schema.org/draft-07/json-schema-validation.html#rfc.section.10.3)).
+
+Basically, in Async API context, it means that a `writeOnly` property makes sense in a **request** only (as password for example).
+
+- `writeOnly` properties are hidden when they belong to a subscribe operation (messages produced by the application and sent to the channel).
+
+`readOnly` makes sense in a **response** only ( `created_at`, `updated_at`, `uuid`...). Similarly, we decided to hide a property defined as readOnly when it belongs to a publish operation (messages consumed by the application from the channel).
 
-Basically, in Async API context, it means that a writeOnly property makes sense in request only (as password for example). Thus, we decided to hide a property defined as writeOnly when it belongs to a subscribe operation (messages produced by the application and sent to the channel).
+- `readOnly` properties are hidden when they belong to a publish operation (messages consumed by the application from the channel).
 
-And a readOnly makes sense in response only (as created_at, updated_at, your call...). Similarly,  we decided to hide a property defined as readOnly when it belongs to a publish operation (messages consumed by the application from the channel).
+:::info
+Not displaying `writeOnly` properties in subscribe operations and `readOnly` properties in publish operations allows the use of same `Schema Object` everywhere it is needed, without generating  confusing informations in the documentation.
+:::
 
-Thus, it becomes easy to use the same Schema Object in different context, for example as seen below:
+Thus, it becomes easy to use the same `Schema Object` in different contexts, for example as seen below:
```

Comment:
Adding the writeOnly and readOnly behaviors makes sense, but maybe it's too dense here. Do you think we could simplify it?

Also, here are some specific feedbacks:
* typo: `Async API` VS `AsyncAPI`
* using the request/response paradigm is a bit surprising in the context of async apis (as there are no requests/responses but events)
* as the writeOnly/readOnly statement is not related to AsyncAPI specifically, should we start extracting this in a `JSON schema` page, and point to this page from OpenAPI and AsyncAPI ones?

Would you like some help with simplifying the block? We can work together on this :)

Reply:
All of the above has been addressed in https://github.com/bump-sh/docs/pull/72/commits/8fc67bc3ed3b722ddd2ced518e927dc57ed31c08. I've extracted the content from openapi and asyncapi support in a JSON schema page and simplified so it makes sense in both OpenAPI and AsyncAPI context. Let me know your thoughts or feedbacks